modules/version-control/version_control_exercise1.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Case Study
+type: exercise
+order: 2
+---
+
+## Exercise: Case Study
+
+**Case study**: A researcher comes to you with a problem of how to keep track of different versions of code. 
+
+What advice would you give the researcher to keep track of changes of their scripts/code and when best to do this (objective 1, 2, 3 and 4). 
```

Comment:
Is this a reference to specific objectives discussed elsewhere or is this more "structure your answer as a list of objectives"?

Reply:
I think it's the first. I will take it out as it is not interesting to the learner.